package.json

```diff
@@ -27,34 +27,38 @@
   },
   "dependencies": {
     "prop-types": "^15.5.10",
-    "styled-components": "^2.1.1"
+    "styled-components": "^4.1.3"
```

Comment:
The things you were using in styled-components didn't require a change, but I figured I could update this package while I was updating the other ones.